src/OpenTelemetry.Exporter.Prometheus/Implementation/PrometheusExporterEventSource.cs

```diff
@@ -72,5 +72,14 @@ public void FailedShutdown(string exception)
         {
             this.WriteEvent(3, exception);
         }
+
+        [Event(4, Message = "No metrics are available for export.", Level = EventLevel.Warning)]
+        public void NoMetrics()
+        {
+            if (this.IsEnabled(EventLevel.Warning, EventKeywords.All))
```

Comment:
we don't need this check. Its done automatically.
(The only time we do that check is to prevent doing some work (like ex.ToString()), if that LogLevel is not enabled)

Reply:
Removed extra check for `EventLevel.Warning`.